docs/src/app/components/pages/customization/themes.md

```diff
@@ -18,10 +18,14 @@ import MuiThemeProvider from 'material-ui/lib/MuiThemeProvider';
 import getMuiTheme from 'material-ui/lib/styles/getMuiTheme';
 import AppBar from 'material-ui/lib/app-bar';
 
+// This replaces all text colors with cyan500
```

Comment:
the theme is calculated only from baseTheme. muiTheme is override of the calculated values. But you're right I should do better with this. :+1: